frontend/packages/webterminal-plugin/console-extensions.json

```diff
@@ -7,9 +35,14 @@
     }
   },
   {
-    "type": "console.flag/hookProvider",
+    "type": "console.page/route/standalone",
     "properties": {
-      "handler": { "$codeRef": "useWebTerminalProvider.useWebTerminalProvider" }
+      "path": "/terminal",
+      "component": { "$codeRef": "CloudShellTab" },
+      "exact": false
+    },
+    "flags": {
+      "required": ["DEVWORKSPACE"]
```

Comment:
Note that instead of redirecting to `/` this will cause `/terminal` to produce a 404 when the webterminal operator is not present. 

I think this is a better side effect to be honest as I think this may be the only place where we were redirecting to `/` on a "not found" scenario